cs_scanimage.py

```diff
@@ -141,33 +145,40 @@ def export(self, filename):
     # loop through and find high severity vulns
     # return HighVulnerability enum value
     def get_alerts_vuln(self):
-        print("running get_alerts_vuln")
+        log.info("running get_alerts_vuln")
         vuln_code = 0
         vulnerabilities = self[self.vuln_str_key_1]
         if vulnerabilities is not None:
             for vulnerability in vulnerabilities:
-                try:
-                    severity = vulnerability[self.vuln_str_key_2][self.details_str_key][self.cvss_str_key][self.sev_str_key]
-                    if severity.lower() == self.severity_high:
-                        vuln_code = ScanStatusCode.HighVulnerability.value
-                        print("Alert: High severity vulnerability found")
-                        break
-                except KeyError:
-                    continue
+                vuln = vulnerability['Vulnerability']
+                cve = vuln.get('CVEID', 'CVE-unknown')
+                details = vuln.get('Details', {})
+                cvss_v3 = details.get('cvss_v3_score', {})
+                severity = cvss_v3.get('severity')
+                if severity is None:
+                    cvss_v2 = details.get('cvss_v2_score', {})
+                    severity = cvss_v2.get('severity')
+                if severity is None:
+                    severity = details.get('severity', 'UNKNOWM')
```

Comment:
@isimluk @redhatrises  typo here. Opened https://github.com/CrowdStrike/container-image-scan/pull/18 to address.